instrumentation/kafka-clients/kafka-clients-0.11/library/src/main/java/io/opentelemetry/instrumentation/kafkaclients/SpanDecorator.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.kafkaclients;
+
+import io.opentelemetry.api.trace.Span;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.apache.kafka.clients.producer.ProducerRecord;
+
+public interface SpanDecorator {
```

Comment:
Is this supposed to be a means of customization for the library instrumentation? You can use the `AttributesExtractor` interface instead - check out one of our [tracing builders](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/okhttp/okhttp-3.0/library/src/main/java/io/opentelemetry/instrumentation/okhttp/v3_0/OkHttpTracingBuilder.java#L38) for an example of that.
Basically it allows the users of the library instrumentations to customize the telemetry generated by it in a standardized fashion (i.e. all our library instrumentations use `AttributesExtractor` for that purpose)

Reply:
OK, this is a copy/paste from OpenTracing, to provide equivalent behavior / API .
Let me see how this `AttributesExctractor` can be used instead ...